flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobServerDeleteTest.java

```diff
@@ -251,6 +251,104 @@ private void testDeleteTransientFails(@Nullable final JobID jobId) throws IOExce
         }
     }
 
+    @Test
+    public void testDeletePermanent() throws IOException {
```

Comment:
Is it possible to reuse the code of `testDeleteTransient`, `testDeleteTransientAlreadyDeleted` and `testDeleteTransientFails`?